cmd/kubeadm/app/cmd/reset.go

```diff
@@ -185,8 +183,7 @@ func resetConfigDir(configPathDir, pkiPathDir string) {
 	}
 	fmt.Printf("[reset] Deleting files: %v\n", filesToClean)
 	for _, path := range filesToClean {
-		err := os.RemoveAll(path)
-		if err != nil {
+		if err := os.RemoveAll(path); err != nil {
```

Comment:
I prefer initializations in ifs where possible since it reduces scope pollution by limiting visibility of the initialized variables.